openy_calc/openy_calc.module

```diff
@@ -48,3 +71,74 @@ function openy_calc_preprocess_fieldset(&$variables) {
     $legend_span->setAttribute('tabindex', '0');
   }
 }
+
+/**
+ * Remove branch_hours from the related branch for "facility" node viewing.
+ */
+function openy_calc_preprocess_node__branch__calc_summary(&$variables) {
+  $node = $variables['node'];
+  if(!$node instanceof NodeInterface) {
+    return;
+  }
+
+  $variables['branch'] = [
+    'branch_name' => [
+      '#type' => 'inline_template',
+      '#template' => '<h4>{{ branch_name }}</h4>',
+      '#context' => [
+        'branch_name' => $node->label(),
+      ],
+    ],
+    'branch_phone' => [
+      '#type' => 'inline_template',
+      '#template' => '<a href="tel:{{ branch_phone }}"><i class="fa fa-phone" aria-hidden="true"></i><div>{{ branch_phone }}</div></a>',
+      '#context' => [
+        'branch_phone' => $node?->get('field_location_phone')->value,
+      ],
+    ],
+    'branch_fax' => [
+      '#type' => 'inline_template',
+      '#template' => '{% if branch_fax  %}<a href="tel:{{ branch_fax }}"><i class="fa fa-phone" aria-hidden="true"></i><div>{{ branch_fax }}</div></a> {% endif %}',
+      '#context' => [
+        'branch_fax' => $node?->get('field_location_fax')->value,
```

Comment:
same as https://github.com/open-y-subprojects/openy_custom/pull/76/files#r1694709572

Reply:
On line 80 we have condition $node, which it  instanceof NodeInterface so $node canot be NULL